lib/java/gradle/environment.gradle

```diff
@@ -66,7 +67,8 @@ dependencies {
     compile "org.slf4j:slf4j-api:${slf4jVersion}"
     compile "org.apache.httpcomponents:httpclient:${httpclientVersion}"
     compile "org.apache.httpcomponents:httpcore:${httpcoreVersion}"
-    compile "javax.servlet:servlet-api:${servletVersion}"
+    //compile "javax.servlet:servlet-api:${servletVersion}"
+    compile "org.apache.tomcat.embed:tomcat-embed-core:${tomcatEmbedVersion}"
```

Comment:
Was "testCompile" meant here? As is, this becomes a dependency downstream.

[THRIFT-5375](https://issues.apache.org/jira/browse/THRIFT-5375).